.travis.yml

```diff
@@ -84,4 +84,4 @@ jobs:
     script:
     - pycodestyle --ignore=E402,W503,W504 lib/cylc lib/Jinja2Filters/*.py lib/parsec/*.py $(grep -l '#!.*\<python\>' bin/*)
     - PYTHONPATH=$(pwd -P)/lib/ pytest --cov-append --cov=lib/cylc --cov=lib/parsec
-    after_script: true
+    after_script: true
```

Comment:
Reset this change?

Reply:
This one is probably my fault, during rebase conflict resolution. (It looks familiar).